config/webpack.config.js

```diff
@@ -84,7 +87,170 @@ function getEntryPoints(entry) {
   return getWebpackEntryPoints(manifestsToBuild);
 }
 
-module.exports = (env = {}) => {
+async function generateHtmlFiles(buildPath) {
+  const TMP_SCAFFOLD_PATH = 'tmp/scaffold';
+  fs.removeSync(TMP_SCAFFOLD_PATH);
+  fs.ensureDirSync(TMP_SCAFFOLD_PATH);
```

Comment:
Do we still need to make the `tmp` directory if the `content-build` directory exists locally? I noticed that the `tmp` directory with all the downloaded files are generated, despite having `content-build` cloned locally.

Reply:
Probably not, but for the sake of keeping the code simple and the flow consistent, I figured it should be fine to read from the same place.